components/raftstore/src/store/snap/io.rs

```diff
@@ -113,9 +113,16 @@ where
 {
     let mut sst_writer = create_sst_file_writer::<E>(engine, cf, path)?;
     let mut stats = BuildStatistics::default();
+    let mut remained_quota = 0;
     box_try!(snap.scan_cf(cf, start_key, end_key, false, |key, value| {
         let entry_len = key.len() + value.len();
-        io_limiter.blocking_consume(entry_len);
+        while entry_len > remained_quota {
+            // It's possible to acquire more than necessary, but let it be.
+            io_limiter.blocking_consume(IO_LIMITER_CHUNK_SIZE);
```

Comment:
This PR makes them fair by underestimating the generating traffic and overestimating the receiving traffic? What if there is only generating traffic? The real generating traffic will be limited far below what we actually want.

Reply:
No, it just make them share the I/O limiter fairly. Without the PR snapshot receiving will acquire more bytes from the limiter, which will slow down generating.